### PR TITLE
Add ChatGPT Tutorial

### PR DESCRIPTION
**Pull request by danielogen**
This PR provides the initial work on adding ChatGPT tutorial to CS472

### CODE REVIEW

**Comment by danielogen** [comment]
> WIP, should I wait to merge it yet?

I am adding some few more content then it will be ready for merge

**Comment by danielogen** [comment]
Hello @johnxu21, 
I think the PR looks good for now. Let me know of any area for improvement

**Comment by johnxu21** [comment]
I think I will only be able to test it after it has been deployed. Let me merge it and then let you know

**Comment by danielogen** [comment]
> I think I will only be able to test it after it has been deployed. Let me merge it and then let you know

Alright

**Review by johnxu21 — CHANGES_REQUESTED** [review]

File: teaching/CS472/Timetable/GPT/gpt.md
[the remark this answers is not in the document]
Comment (johnxu21) [reply teaching/CS472/Timetable/GPT/gpt.md:1]: Good one, lets get a second example

**Review by johnxu21 — CHANGES_REQUESTED** [review]

File: teaching/CS472/Timetable/GPT/gpt.md
[the remark this answers is not in the document]
Comment (danielogen) [reply teaching/CS472/Timetable/GPT/gpt.md:1]: I have made some update following this suggestion

**Pull request merged**